CMakeLists.txt

```diff
@@ -565,3 +565,11 @@ if(WIN32)
           RUNTIME DESTINATION lib/$<CONFIG>
           ARCHIVE DESTINATION lib/$<CONFIG>)
 endif()
+
+message(STATUS "summary of build options:
```

Comment:
This probobly can be removed, I use this to check the compiler used is intended https://github.com/gengjiawen/libuv/runs/671071718?check_suite_focus=true#step:5:46

```console
-- summary of build options:
    Install prefix:  /usr/local
    Target system:   Linux
    Compiler:
      C compiler:    /usr/bin/arm-linux-gnueabi-gcc
      CFLAGS:      
```

Reply:
Seems pretty useful to me. I don't mind keeping it.